dashboard/test/ui/features/star_labs/ailab.feature

```diff
@@ -0,0 +1,67 @@
+Feature: AI Lab
+
+  @as_student
+  Scenario: Training a model with an in-house dataset and importing it into App Lab
+    Given I am on "http://studio.code.org/s/allthethings/lessons/43/levels/1"
+    And I wait to see ".editor-column"
+    And I see the dynamic instructions are showing "uitest-dynamic-instruction-selectDataset" key
+    And I click selector "#overlay"
+    And I click selector ".uitest-ailab-dataset-image"
```

Comment:
This feels ambiguous because you're clicking one item but presumably this class applies to many.

Reply:
For `.editor-column`, it looks like there are other divs with that class on the page. It also looks like it's used in other areas of the [codebase](https://github.com/code-dot-org/code-dot-org/search?q=%22.editor-column%22). 

For `.uitest-ailab-dataset-image`, there are multiple items with this class that can be clicked on. We could change these to IDs that incorporate the dataset name, but then the tests would be dependent on that specific dataset. I'd prefer to keep this dataset-agnostic. Since the corresponding code-dot-org/ml-playground#274 was merged and a build was cut for this, this could be handled in a future PR.